charts/redpanda/templates/configmap.yaml

```diff
@@ -66,7 +66,7 @@ metadata:
 data:
 {{- if (include "redpanda-atleast-22-1-1" . | fromJson).bool }}
   bootstrap.yaml: |
-    enable_sasl: {{ dig "sasl" "enabled" false .Values.auth }}
```

Comment:
what happens if both are available? and in which version of rp did the change occur?

Reply:
The change was introduced in https://github.com/redpanda-data/redpanda/pull/5292 PR. That change was released in https://github.com/redpanda-data/redpanda/releases/tag/v22.2.1. According to PR cover letter `kafka_enable_authorization` overwrites the `enable_sasl`. They can coexist.